src/components/DocLink.tsx

```diff
@@ -1,94 +1,66 @@
 import React from "react"
 import { useTheme } from "@emotion/react"
-import styled from "@emotion/styled"
-import Icon from "./Icon"
+import { Icon, Box, Container, Text } from "@chakra-ui/react"
+import { AiOutlineArrowRight } from "react-icons/ai"
 import Link from "./Link"
-import Emoji from "./OldEmoji"
+import Emoji from "./Emoji"
 
 export interface IProps {
   children?: React.ReactNode
   to: string
-  className?: string
   isExternal?: boolean
 }
 
-const Container = styled(Link)<{ isExternal: boolean }>`
-  position: relative;
-  z-index: 1;
-  text-decoration: none;
-  display: flex;
-  flex-direction: row;
-  flex: 1;
-  width: 100%;
-  justify-content: space-between;
-  padding: 1rem;
-  border-radius: 2px;
-  color: ${({ theme }) => theme.colors.text};
-  border: 1px solid ${({ theme }) => theme.colors.border};
-  &:hover {
-    text-decoration: none;
-    box-shadow: 0 0 1px ${({ theme }) => theme.colors.primary};
-    background: ${({ theme }) => theme.colors.tableBackgroundHover};
-    border-radius: 4px;
-
-    svg {
-      fill: ${(props) => props.theme.colors.primary};
-      transition: transform 0.1s;
-      transform: scale(1.2)
-        rotate(${({ isExternal }) => (isExternal ? "-45deg" : "0")});
-    }
-  }
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
-    width: 100%;
-  }
-`
-
-const TextCell = styled.div`
-  flex: 1;
-  flex-direction: column;
-  color: ${({ theme }) => theme.colors.text};
-`
-
-const Title = styled.p`
-  color: ${({ theme }) => theme.colors.text300};
-  font-weight: 600;
-  margin: 0;
-`
-
-const Arrow = styled(Icon)<{ isExternal: boolean }>`
-  margin: 0rem 1.5rem;
-  align-self: center;
-  min-width: 2rem;
-  transform: rotate(${({ isExternal }) => (isExternal ? "-45deg" : "0")});
-`
-
-const EmojiCell = styled.div`
-  display: flex;
-  align-items: center;
-`
-
-const DocLink: React.FC<IProps> = ({
-  to,
-  children,
-  className,
-  isExternal = false,
-}) => {
+const DocLink: React.FC<IProps> = ({ to, children, isExternal = false }) => {
   const theme = useTheme()
 
   return (
-    <Container to={to} className={className} isExternal={isExternal} hideArrow>
-      <EmojiCell>
-        <Emoji size={1} text=":page_with_curl:" mr={`1rem`} />
-      </EmojiCell>
-      <TextCell>
-        <Title>{children}</Title>
-      </TextCell>
-      <Arrow
-        isExternal={isExternal}
-        color={theme.colors.text}
-        name="arrowRight"
-      />
-    </Container>
+    <Link to={to} isExternal={isExternal} textDecoration="none" hideArrow>
```

Comment:
Lets try to use https://chakra-ui.com/docs/components/link-overlay/usage. I have used in this other PR #7721

Reply:
Wouldn't it be better to use the ChakraUI <Link> component instead of <LinkBox> and <LinkOverlay> since there aren't going to be multiple links inside the component?

Reply:
Not sure if I follow you on this one. We don't have multiple links in this component.

What `LinkOverlay` does is avoid wrapping the component with a `<a>` tag (as we were doing) which is not semantically correct. It would be an improvement on what we had.

Reply:
Ohhhhh ok. That makes sense. Sorry if my questions seem silly. It's my first time using ChakraUI actually so thanks for being patient with me and giving me suggestions.

Reply:
That is totally fine! I didn't explain myself correctly. Glad that you brought that concern up 💯